masterlist.yaml

```diff
@@ -2576,6 +2576,13 @@ plugins:
       - name: 'RLO - Interiors.esp'
         condition: 'not file("Realistic Lighting Overhaul - Major City Interiors.esp")'
 
+  - name: 'ENB Light.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/22574' ]
+    after:
+      - 'ELE_SSE.esp'
+    req:
+      - 'Particle Patch for ENB SSE.esp'
```

Comment:
My only concern with the req is the plugin `Particle Patch for ENB SSE.esp` has been superceded by [ENB Helper](https://www.nexusmods.com/skyrimspecialedition/mods/23174). the meshes and texture from particle patch can still be used with ENB helper. But it's not major as users can still disable `Particle Patch for ENB SSE.esp` if using ENB Helper and the requirement will be fufilled.